src/Rootfs.jl

```diff
@@ -471,6 +472,12 @@ function gcc_version(p::AbstractPlatform,GCC_builds::Vector{GCCBuild},
         GCC_builds = filter(b -> getversion(b) ≥ v"7", GCC_builds)
     end
 
+    # We only have GCC 12 or newer for RISC-V.
+    # (This could be changed down to GCC 7.1.)
```

Comment:
I think it's fine to have only recent toolchains for recently introduced platforms, it reduces the maintenance burden.

Reply:
Ok.